src/main/java/net/minecraftforge/client/model/ForgeFaceData.java

```diff
@@ -28,11 +28,12 @@
  * @param blockLight Block Light for this face from 0-15 (inclusive)
  * @param skyLight Sky Light for this face from 0-15 (inclusive)
  * @param ambientOcclusion If this face has AO
+ * @param calculateNormals If we should manually calculate the normals for this block or inherit facing normals like vanilla
  */
-public record ForgeFaceData(int color, int blockLight, int skyLight, boolean ambientOcclusion)
+public record ForgeFaceData(int color, int blockLight, int skyLight, boolean ambientOcclusion, boolean calculateNormals)
 {
 
-    public static final ForgeFaceData DEFAULT = new ForgeFaceData(0xFFFFFFFF, 0, 0, true);
+    public static final ForgeFaceData DEFAULT = new ForgeFaceData(0xFFFFFFFF, 0, 0, true, false);
```

Comment:
bincompat: Add a constructor without the boolean parameter and give it the default value of `false`.